src/pybind/mgr/volumes/module.py

```diff
@@ -49,8 +49,20 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         {
             'cmd': 'fs volume create '
                    f'name=name,type=CephString,goodchars={goodchars} '
-                   'name=placement,type=CephString,req=false ',
-            'desc': "Create a CephFS volume",
+                   'name=placement,type=CephString,req=false '
+                  f'name=meta_pool,type=CephString,goodchars={goodchars},req=false '
+                  # XXX goodchars here is not being passed deliberately since
+                  # --data-pool can receive a list of data pool names separated
+                  # by commas and that is a problem. If goodchars variable
+                  # declared in this module is passed as it is(that is without
+                  # adding comma to it), arg parser would reject comma separated
+                  # data pool names. And if comma is added to goodchars and then
+                  # passed below, arg parser misinterprets it since comma has a
+                  # special # meaning for it. So the best option is to not
+                  # restrict data pool names to goodchars and do that
+                  # restriction elsewhere.
+                  f'name=data_pool,type=CephString,req=false ',
+            'desc': "Create a CephFS volume, create pools too if not passed",
```

Comment:
'desc': "Create a CephFS volume. Pools will be created as necessary if specified."

Reply:
Fixed.